vscode/src/editor/vscode-editor.ts

```diff
@@ -139,7 +138,7 @@ export class VSCodeEditor implements Editor<InlineController, FixupController, C
             return null
         }
         const selection = activeEditor.selection
-        if (!selection?.start.line) {
+        if (!selection?.start.line && selection?.start.line !== 0) {
```

Comment:
Two negations are confusing, so you could have `!(... || ...)`. But in this case you can be direct by just thinking about the types: It is undefined, or a number. So check if that is the case.

I can't remember the specific context but I've seen this mistake before. Something to add to your test cases and personal desk check, whether it handles working on line 0.
 
```suggestion
        if (selection?.start.line === undefined) {
```

Reply:
Oh yea I could just use `!selection.start` 😆  Thank you for your reviews as always 🙇‍♀️